hermes/bin/src/runtime_extensions/wasi/clocks/monotonic/host.rs

```diff
@@ -11,12 +12,12 @@ impl Host for HermesState {
     /// The clock is monotonic, therefore calling this function repeatedly will
     /// produce a sequence of non-decreasing values.
     fn now(&mut self) -> wasmtime::Result<Instant> {
-        todo!()
+        now_impl()
     }
 
     /// Query the resolution of the clock. Returns the duration of time
     /// corresponding to a clock tick.
     fn resolution(&mut self) -> wasmtime::Result<Duration> {
-        todo!()
+        resolution_impl()
```

Comment:
Because of the above, this is always nanoseconds, and needs not be any other value.